mmv1/third_party/terraform/tests/data_source_dns_record_set_test.go.erb

```diff
@@ -0,0 +1,108 @@
+<% autogen_exception -%>
+package google
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
```

Comment:
We expect to swap in the new test package instead of relying on the plugin SDK for these when we get the chance, right? https://github.com/hashicorp/terraform-plugin-testing

(not needed to solve in this PR, I mean in general)